### PR TITLE
Add links to inventory

### PR DESCRIPTION
**Pull request by cgebken**

### CODE REVIEW

File: beyond-essence/inventory/links.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Position
+
+Links are positioned in various locations throughout the software.
+They are displayed
+
+- on cards,
+- in the topbar,
+- and in tables.
+
```
Comment (vzerwas) [inline beyond-essence/inventory/links.md]: Links can also occur in settings layers

File: beyond-essence/inventory/links.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+{% image_custom image="/assets/img/pages/essence/links-breadcrumbs.png" width="50" lightbox %}
+
+* Links embedded in text
+
+{% image_custom image="/assets/img/pages/essence/links-embedded.png" width="50" lightbox %}
+
+* Pagination links
+
+{% image_custom image="/assets/img/pages/essence/links-pagination.png" width="50" lightbox %}
+
```
Comment (vzerwas) [inline beyond-essence/inventory/links.md]: Just for completeness, we also have only icons as links (Online Shop overview page):
Bildschirmfoto 2019-08-09 um 14 33 55

File: beyond-essence/inventory/links.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Links are navigation elements.
+They are interactive and give users access to further content that is not directly accessible via the primary navigation.
+
+## Use cases
+
+In the Beyond cockpit, we provide various types of links:
+
+* Links in tables and overviews
+
+{% image_custom image="/assets/img/pages/essence/links-in-tables.png" width="50" lightbox %}
```
Comment (vzerwas) [inline beyond-essence/inventory/links.md]: Proposal: We could distinguish a little bit more. Described example above is an example for `Links in tables` and we also could have a section `Links in overviews`:
Bildschirmfoto 2019-08-09 um 14 36 26
I know from a technical point of view both examples are tables so it is more a visual perception.

File: beyond-essence/inventory/links.md (hunk 2 above)
Comment (cgebken) [reply beyond-essence/inventory/links.md]: As already discussed: This is definitely a topic for later. If we sit together, let's discuss if we'd like to add these linked icons here or in the icons article.

Comment (vzerwas) [reply beyond-essence/inventory/links.md]: I took a note ;-)

File: beyond-essence/inventory/links.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- in the topbar,
+- in settings layers,
+- in modal dialogs,
+- in overviews,
+- and in tables.
+
+## Behavior
+
+Clicking or touching a link will open new content, e.g. a view, a popup, or a PDF document.
+Depending on the use case or implementation, the content will be opened in a new tab, in the same tab, or in a new tool.
```
Comment (katjamueller) [inline beyond-essence/inventory/links.md]: What do you mean with 'new tool'?

File: beyond-essence/inventory/links.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+title: Links
+background_image: index.jpg
+---
+
+Links are navigation elements.
+They are interactive and give users access to further content that is not directly accessible via the primary navigation.
+
+## Use cases
+
+In the Beyond cockpit, we provide various types of links:
```
Comment (katjamueller) [inline beyond-essence/inventory/links.md]: I would write something like:
> In the Beyond cockpit we provide two types of links:

Primary link
Default link

They vary in the form that they are used with icons or without. An icon before the link supports the content. A link without icon is used for self explained links.

Comment (cgebken) [reply beyond-essence/inventory/links.md]: I've moved the "old" use cases to the "position" section and added the primary and secondary links as new use cases. 

When having a look at the primary/secondary issue again, I also figured out that I need to extend the "structure" and "behavior" section a bit.

**Pull request merged**